### PR TITLE
Add events feed endpoint

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

File: src/nyc_trees/apps/event/views.py [hunk 1]
```diff
@@ -166,8 +166,38 @@ def events_list_page_partial(request):
...
+            "address": e.address,
+            # NOT POSSIBLE WITHOUT REVERSE GEOCODING:
+            # "zip": None,
+            # WHERE WOULD WE GET THESE? PROBABLY NEED TO DROP:
+            # "park_id": None,
+            # "borough": None,
+            # "accessible": None,
+        }],
+        # EVENT HAS NO LINKS. WALK UP TO event.group.contact_url ?:
+        # 'links': [],
```
Comment (maurizi) [inline src/nyc_trees/apps/event/views.py]: I think we should link to the URL for the event detail here.

File: src/nyc_trees/apps/event/views.py [hunk 2]
```diff
@@ -166,8 +166,38 @@ def events_list_page_partial(request):
...
+        # NOT INCLUDED IN THIS APP. DROPPED:
+        # 'phone': None,
+        # 'cost_free': None,
+        # 'cost_description': None,
+        # 'categories': [],
+    } for e in Event.objects.order_by('-begins_at')]
```
Comment (maurizi) [inline src/nyc_trees/apps/event/views.py]: I think we should exclude private events from this list.

**Comment by dboyer** [comment]
Yes, we should exclude private events.

**Comment by maurizi** [comment]
I think we should change the URL for this view to end in `.json`

**Comment by maurizi** [comment]
+1, after fixing my the comment about the URL

**Pull request merged**